src/components/BackstageComparison/FeatureComparisonTable.js

```diff
@@ -0,0 +1,413 @@
+import React, { Fragment } from 'react';
+import { CheckIcon, MinusIcon } from '@heroicons/react/solid';
+import { TextLink } from 'components';
+
+import COMPETITORS from './competitors';
+
+const sections = [
+  {
+    name: 'Software catalog',
+    features: [
+      {
+        name: 'Bring your own data model',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+      {
+        name: 'Entity definition',
+        competitor: { Roadie: 'Markdown, UI, or via API', Backstage: 'Markdown' },
```

Comment:
What does "Entity definition" mean in this context? I actually don't understand.